src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,152 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.exception.DuplicateUserException;
+import core.basesyntax.exception.InvalidAgeException;
+import core.basesyntax.exception.InvalidLoginStringException;
+import core.basesyntax.exception.InvalidStringBetException;
+import core.basesyntax.exception.InvalidStringRegisterException;
+import core.basesyntax.exception.UserNotAuthorizedException;
+import core.basesyntax.exception.WrongLoginOrPasswordException;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import core.basesyntax.service.AuthorizationService;
+import core.basesyntax.service.Mapper;
+import core.basesyntax.service.RegistrationService;
+import java.util.Scanner;
+import lombok.Setter;
+
+public class ConsoleHandler {
+    private static final String CURSOR_TEXT = "\nNeed to log in [/l] or register [/r]. Help [/h]->";
+    @Inject
+    private BetDao betDao;
+    @Setter
+    private AuthorizationService authService;
+    @Setter
+    private RegistrationService regService;
+    @Setter
+    private Mapper<User> userMapper;
+    @Setter
+    private Mapper<Bet> betMapper;
+
+    public void handle() {
+        String cursor = CURSOR_TEXT;
+        User user = null;
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.print(cursor);
+            String line = scanner.nextLine();
+            String[] data = line.split(" ");
+            try {
+                switch (data[0]) {
+                    case "/q":
+                        return;
+                    case "/l":
+                        user = authorize(data);
+                        cursor = "\n[" + user.getLogin() + "] enter bet->";
+                        continue;
```

Comment:
I think you can call `break` instead of `continue` in the end of each `case`
this is the usual case how we should write `switch` block